UITests/ClearPrivateDataTests.swift

```diff
@@ -33,7 +38,10 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         tester().waitForAbsenceOfViewWithAccessibilityLabel("Clear Everything")
 
         tester().tapViewWithAccessibilityLabel("Done")
-        tester().tapViewWithAccessibilityLabel("home")
+        // on the ipad air sometimes we will find ourselves already out of the tab tray so no need to click 'home'
+        if tester().tryFindingViewWithAccessibilityLabel("home", error: nil) {
+            tester().tapViewWithAccessibilityLabel("home")
+        }
```

Comment:
We have some code on creating a new tab that says "If there is a modal view controller in front of me, don't pop the tab tray when adding a new tab". This is to prevent a situation where the tab tray animation doesn't occur because the tab tray controller is in the background leaving the newly created tab with dodgy UI. 
When executed on an iPad, however, the Tab Tray Controller is still active as the modal is a form factor type, which means that the tab tray is "popped" and the home tab is automatically selected. Without this check then the test will always fail when executed on an iPad as there will be no "home" tab tray collection view cell to tap on.